plugins/default/lib/test_nagios.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+import unittest
+
+import nagios
+
+
+class TestParseResponse(unittest.TestCase):
```

Comment:
Normally I might agree, but these are very important to **nagios**. To me if something is being tested it means it's important and might make you think twice if you go changing it.